src/main/java/net/sf/jabref/importer/fileformat/MedlinePlainImporter.java

```diff
@@ -141,20 +143,17 @@ public boolean isRecognizedFormat(InputStream stream) throws IOException {
                 String entry = current.toString();
 
                 String lab = entry.substring(0, entry.indexOf('-')).trim();
-                String val = entry.substring(entry.indexOf('-') + 1).trim();
+                String value = entry.substring(entry.indexOf('-') + 1).trim();
+
                 if ("PT".equals(lab)) {
-                    val = val.toLowerCase();
+                    String val = value.toLowerCase(Locale.ENGLISH);
```

Comment:
You should do a general renaming refactoring, it is hard to follow statements like this and with all the abbreviations, hm, lab, val,...